src/Features/CSharp/Portable/SplitStringLiteral/InterpolatedStringSplitter.cs

```diff
@@ -25,11 +26,11 @@ private sealed class InterpolatedStringSplitter : StringSplitter
                 SyntaxNode root,
                 SourceText sourceText,
                 InterpolatedStringExpressionSyntax interpolatedStringExpression,
+                IndentationOptions options,
                 bool useTabs,
                 int tabSize,
```

Comment:
Not exactly. These are read from the editor options while the IndentationOptions are read from editorconfig or solution. This should be consolidated once we have APIs from the editor that would allow us to read all options from the editor.